src/pages/en/core-concepts/partial-hydration.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Partial Hydration in Astro
+description: Learn how partial hydration works using the "Islands Architecture" in Astro.
+---
+
+**Astro generates every website with zero client-side JavaScript, by default.** Use any frontend UI component that you'd like (React, Svelte, Vue, etc.) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.
```

Comment:
Can you link to these frameworks websites, similar to how you do in the other page?

Reply:
Done!